.github/workflows/clp-artifact-build.yaml

```
@@ -1,28 +1,14 @@
   push:
-    paths:
-      - ".github/actions/clp-core-build-containers/action.yaml"
-      - ".github/actions/run-on-image/action.yaml"
-      - ".github/workflows/clp-core-build.yaml"
-      - ".gitmodules"
-      - "components/core/**"
-      - "taskfile.yaml"
-      - "taskfiles/**"
-      - "tools/scripts/deps-download/**"
-      - "!components/core/tools/scripts/lib_install/macos/**"
+    paths-ignore: *ignored_paths
```

actionlint (1.7.7)

"paths-ignore" section must be sequence node but got alias node with "" tag

(syntax-check)